tracer/src/Datadog.Trace/Agent/Transports/HttpStreamRequest.cs

```diff
@@ -48,8 +52,22 @@ public async Task<IApiResponse> PostAsJsonAsync(IEvent events, JsonSerializer se
             {
                 serializer.Serialize(writer, events);
                 await writer.FlushAsync();
+                await memoryStream.FlushAsync();
```

Comment:
I think these should have `ConfigureAwait(false)` too?

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1778/commits/62c946f8890c5547eb463e90479a50f783e369bf